rangecoder.h

```diff
@@ -217,11 +217,9 @@ namespace rangecoder
     class RangeDecoder : RangeCoder
     {
     public:
-        RangeDecoder() = delete;
-
-        RangeDecoder(std::queue<byte_t> bytes)
+        void start(std::queue<byte_t> bytes)
```

Comment:
rename to initialize
```suggestion
        void initialize(std::queue<byte_t> bytes)
```